### ISSUE TITLE
OCA JSON file created outside of directory 

### ISSUE THREAD

**Issue by RodneyBaker** [opentoonz/opentoonz#5505]
### Description

The location of the OCA JSON file in Opentoonz (and Tahoma2D) is currently created outside the OCA directory containing the assets of the project.   This has some potential problems and doesn't meet the OCA specs.  REF:  https://oca.rxlab.guide/specs/folder-structure.html

```Data file
The OCA data file must be at the root of the OCA folder, and should have the same name.

There must be a single OCA data file in the folder, and it must be at the root.
```

### Steps to Reproduce

1.   Export a Scene from Opentoonz via File > Export OCA
2.   Note the location where the JSON file is created.

While not particularly problematic in importing and exporting OCA failure to place the JSON file appropriately will lead to the JSON file being separated from OCA contents when the directory is moved or zipped up.

### Expected Behavior

JSON file should be created inside the OCA directory according to the OCA spec.

**Comment by RodneyBaker** [comment]
Note:  There is nothing that prevents a user from placing the JSON file elsewhere (anywhere they want really) and then fixing the paths to resources in that file to point to the images correctly but adhering to the specs will solve many issue that will rise due to not adhering to the placement of JSON file according to the spec.

**Comment by RodneyBaker** [comment]
Resolution of this issue is implemented for Tahoma2D.
Please see:  https://github.com/tahoma2d/tahoma2d/pull/1520

**Comment by RodneyBaker** [comment]
This problem should be fixed (and enhanced) by #5515 .
